base/mapiterator.jl

```diff
@@ -0,0 +1,175 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+
+"""
+    MapIterator(f, c...) -> iterator
+
+`collect(MapIterator(f, c...))` is equivalent to `map(f, c...)`
+"""
+
```

Comment:
Yes, any amount of whitespace, as well as commented lines, will get ignored. But as Tony mentions, stylistically it isn't quite as obvious. Maybe worth a mention in #15136.

Also missing a fullstop as the end of the line.